ups/ap_pipe.table

```diff
@@ -13,6 +13,8 @@ setupRequired(ndarray)
 setupRequired(obs_decam)
 setupRequired(ap_association)
 
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/bin)
```

Comment:
Did you mean `PATH`?

Reply:
I probably did! That's probably why it doesn't work as I thought it should 😆